### PR TITLE
Re Add tutorial page

### PR DESCRIPTION
**Pull request by 0tuedon**
Added back tutorial page based on the discussion  and  updated the previous images to high quality images

### CODE REVIEW

File: src/layout/index.tsx [hunk 1]
```diff
@@ -10,7 +10,7 @@ import Navbar from "@/components/navbar/Navbar";
 const Layout = ({ children }: { children: React.ReactNode }) => {
   const { noRestriction } = useNoContainerLimit();
   const router = useRouter();
-  const isHomePage = router.asPath === "/";
+  const isHomePage = ["/", "/home"].includes(router.asPath);
```
[the remark this answers is not in the document]
Comment (0tuedon) [reply src/layout/index.tsx:13]: yh it does and then shows to logos and all of that, a fix would be we remove the top header

File: src/layout/index.tsx [hunk 2]
```diff
@@ -24,6 +24,7 @@ const Layout = ({ children }: { children: React.ReactNode }) => {
         flexGrow={1}
         py={!isHomePage ? 4 : 0}
         mt={!isHomePage ? 12 : 0}
+        width={"full"}
```
[the remark this answers is not in the document]
Comment (0tuedon) [reply src/layout/index.tsx:27]: Yh figured out the width won't work and added the implementation for the container

File: src/components/navbar/Navbar.tsx [hunk 3]
```diff
@@ -14,13 +14,13 @@ const Navbar = () => {
       bgColor="whiteAlpha.700"
       fontSize="14px"
       isolation="isolate"
-      zIndex={1}
+      zIndex={50}
```
[the remark this answers is not in the document]
Comment (0tuedon) [reply src/components/navbar/Navbar.tsx:17]: most likely yes if we are going with andreas change and the navbar of the in app is the same with /home we would need to increase it else the picture will be flying over

File: src/layout/index.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (0tuedon) [reply src/layout/index.tsx:13]: @Emmanuel-Develops

[the remark this answers is not in the document]
Comment (Emmanuel-Develops) [reply src/layout/index.tsx:13]: try implementing the navbar in the landing page

**Review by Emmanuel-Develops — APPROVED** [review]

**Review by Emmanuel-Develops — APPROVED** [review]

**Pull request merged**